crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -37,6 +41,17 @@ struct TestCase {
     order: Order,
     fee_policy: Policy,
     execution: Execution,
+    expected_score: eth::U256,
+}
+
+// because of rounding errors, it's good enough to check that the expected value
+// is within a very narrow range of the executed value
+fn is_approximately_equal(executed_value: eth::U256, expected_value: eth::U256) -> bool {
```

Comment:
This is used only for unit test the same as `protocol_fee_test_case`, why not to compile it only for the tests? otherwise we have dead code.

Reply:
Although I didn't notice any unused code warning, that makes sense. Added `#[cfg(test)]` if that is the best option here.